content/docs/0.9.x/operate/advanced_install_options/index.md

```diff
@@ -140,9 +145,25 @@ If you want to use an externally hosted MongoDB instead of the MongoDB installed
 
 ```console
 helm upgrade keptn keptn --install -n keptn --create-namespace
---set=control-plane.mongodb.enabled=false,
+--set=control-plane.mongodb.enabled=true,
```

Comment:
Please revert this to `false`. When installing keptn with an external MongoDB we don't want to install our internal MongoDB, so it makes sense that this option is disabled here.